cartographer/mapping_3d/local_trajectory_builder.cc

```diff
@@ -133,9 +133,9 @@ LocalTrajectoryBuilder::AddAccumulatedRangeData(
       adaptive_voxel_filter.Filter(filtered_range_data.returns);
   if (options_.use_online_correlative_scan_matching()) {
     // We take a copy since we use 'initial_ceres_pose' as an output argument.
-    const transform::Rigid3d initial_pose = initial_ceres_pose;
+    const transform::Rigid3d local_pose = initial_ceres_pose;
```

Comment:
In 3D, we do scan-to-submap matching in the submap frame, so this is not a local pose, but something different. `initial_pose_in_submap` maybe similar to `pose_observation_in_submap` below?